### ISSUE TITLE
User list icons

### ISSUE THREAD

**Issue by brianteeman** [joomla-projects/j4adminui#44]
![image](https://user-images.githubusercontent.com/1296369/68474375-1f42cc80-021d-11ea-90ad-137514cf7ef8.png)

These fail massively for accessibility (as does the rest of the template :( )

1. Never use colour alone to indicate anything. There should always be at least TWO visual indicators
2. The difference between the enabled and blocked icon is almost impossible to see

**Comment by kawshar** [comment]
I agree with you. I also think that we need to change the icons here. I will ask the team to design new icons for those.

Thanks

**Comment by marcodings** [comment]
@brianteeman thanks for your detailed feedback on this specific issue #44

>  as does the rest of the template :( 

I hope issues can and will be addressed as separate and specific issues here by those knoledgable. History has taught us that accessibility is a speciality of its own and a where we have limited expertise within the project, this plagued "Atum" as well.

**Comment by brianteeman** [comment]
As the person who wrote a considerable amount of the accessibility related code you don't need to tell me. Sadly no care at all has been taken for accessibility in this template and the lessons learned creating atum have not been learnt. It is impossible and a **complete waste of my time** to create specific issue reports in the current state of the template as it is no where near satisfying the production department requirements even without looking at accessibility.

**Comment by marcodings** [comment]
Sorry to hear its "impossible"/"waste of your time" to be more specific than "its all wrong".  I was not trying to be Condescendant in my remark, apologies if it was perceived as such, i was trying to give recognition to you and others that have the knowledge.

There are/were different people/teams involved over time and the ONE thing i would say about about all involved in Atum or Khosun.. THEY CARE(D). I even now see them reach out to learn and understand, recognising their limitations. As a project we can improve in "securing" knowledge, there is always room for improvement, we are not there yet.

Having made your point Brian lets leave it at that and not needlessly repeat it.

**Comment by brianteeman** [comment]
@marcodings Apology accepted, it did indeed come across as condescending.

I did make an initial report on accessibility over three weeks ago with over ten specific accessibility issues. None of them have been fixed to date hence my comment that doing anything more is a waste of my time.

**Comment by marcodings** [comment]
> None of them have been fixed to date hence my comment that doing anything more is a waste of my time.

As i tried to explain the CI is not working with us.. I'm convinced the issues will get the well deserved attention once CI is running.

**Comment by brianteeman** [comment]
The CI has nothing to do with writing accessible code - thats just a lame excuse

**Comment by C-Lodder** [comment]
Hate to say it, but I told you so: https://forum.joomla.org/viewtopic.php?f=803&t=970614&p=3580350#p3580350

**Comment by kawshar** [comment]
@brianteeman none of the PR works because of the CI build issue. We've fixed many issues in our local environments and we are waiting for the issue to be fixed. Btw, I don't think you give a damn about the issue we are facing right now. You are here to prove the initiative wrong.

**Comment by brianteeman** [comment]
Not at all. I just want to see a template that satisfies its goals.

**Comment by N6REJ** [comment]
@brianteeman from a gestalt point of view won't the design improve accessibility?

**Comment by Hackwar** [comment]
No

**Comment by brianteeman** [comment]
No for the reasons stated in the original post
